packages/common/src/utils/formatUtil.test.ts

```diff
@@ -0,0 +1,151 @@
+import { AUDIO, FixedDecimal, USDC, wAUDIO } from '@audius/fixed-decimal'
+import BN from 'bn.js'
+
+import { BNWei } from 'models/Wallet'
+
+import {
+  formatCurrencyBalance,
+  formatAudio,
+  formatWeiToAudioString,
+  formatPrice,
+  convertFloatToWei,
+  formatNumberString,
+  formatCount,
+  formatBytes,
+  parseWeiNumber
+} from './formatUtil'
+
+describe('formatUtil', function () {
+  it('can format count', function () {
+    expect(formatCount(0)).toBe('0')
+    expect(formatCount(100)).toBe('100')
+    expect(formatCount(1001)).toBe('1K')
+    expect(formatCount(1200)).toBe('1.2K')
+    // expect(formatCount(1239)).toBe('1.23K') fails: rounds up
+    expect(formatCount(10023)).toBe('10K')
+    expect(formatCount(10423)).toBe('10.4K')
+    expect(formatCount(100000)).toBe('100K')
+    expect(formatCount(103313)).toBe('103K')
+    expect(formatCount(1000000)).toBe('1M')
+    expect(formatCount(1234567)).toBe('1.23M')
+    // expect(formatCount(1235567)).toBe('1.23M') fails: rounds up
+    expect(formatCount(100000000)).toBe('100M')
+  })
+
+  it('can format currency balance', function () {
+    const testCases = [
+      { input: 0, expected: '0' },
+      { input: 410.1545, expected: '410.15' },
+      // { input: 123456789.12345, expected: '123456K' } // note: existing method can't handle this size!
+      { input: 452, expected: '452' },
+      { input: 999999, expected: '999K' },
+      { input: 1000, expected: '1000' },
+      { input: 10000, expected: '10K' },
+      { input: 51701570.34108908, expected: '51701K' }
+      // { input 23.4252, expected: '23.42' } // note: existing method incorrectly rounds this instead of truncating!
+    ]
+    for (const { input, expected } of testCases) {
+      expect(AUDIO(input).toShorthand()).toBe(expected)
```

Comment:
This test file also has parity tests for `FixedDecimal`, look here for more recipes!